packages/tectonic-explorer/js/components/data-collection-dialog.tsx

```diff
@@ -26,7 +27,7 @@ export const DataCollectionDialog: React.FC<IProps> = ({ currentDataSample, onCl
 
   const notesEnabled = config.dataSampleColumns.includes("notes");
   // When user is collecting data, notes should show as a separate textfield under the table,
-  // so notes column require special handling. Also, there's no need to display pin id / number.
+  // so notes column require special handling. Also, there"s no need to display pin id / number.
```

Comment:
The single quote to double quote looks like a search/replace mistake :)

Reply:
It is. :) Thanks, I'll fix it later, so I don't need to jump between branches now.

Reply:
It is. :) I'll fix it later, so I don't need to jump between branches now.